charts/newrelic-logging/tests/rbac_test.yaml

```diff
@@ -0,0 +1,48 @@
+suite: test RBAC creation
```

Comment:
Thank you for taking care of adding all these test cases! 🙇

Reply:
You are welcome :)